src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkPIDController;
+import com.revrobotics.CANSparkBase.ControlType;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+
+
+public class Shooter extends SubsystemBase {
+
+    private CANSparkMax m_motor;
+    private SparkPIDController m_pidController;
+    private RelativeEncoder m_encoder;
+
+    private double kP, kD, kFF, kMaxOutput, kMinOutput, maxRPM; 
+    
+        /// add Tunable Number later for ajusting Rpm on the flywheels 
+
+        public Shooter() {  
+            m_motor = new CANSparkMax(1, MotorType.kBrushless);
```

Comment:
We use singleton for each subsystem, which means only one instance for each subsystem. 
You can refer to other subsystem, e.g. intake or climber.